code/modules/organs/internal/_internal.dm

```diff
@@ -85,14 +90,25 @@
 	for(var/process in organ_efficiency)
 		if(!islist(owner.internal_organs_by_efficiency[process]))
 			owner.internal_organs_by_efficiency[process] = list()
-		owner.internal_organs_by_efficiency[process] += src
+		if(src.is_usable())
+			owner.internal_organs_by_efficiency[process] += src
+		else
+			owner.internal_organs_by_efficiency[process] -= src
 
 	for(var/proc_path in owner_verbs)
 		verbs |= proc_path
 
 	if(GetComponent(/datum/component/internal_wound/organic/parenchyma))
 		owner.mutation_index++
 
+/obj/item/organ/internal/proc/handle_organ_eff()
+	owner = src.owner
```

Comment:
I'm not sure I get what this one is doing. Assign owner to owner?

Reply:
It's nothing more than a messy copypasting from other proc, does absolutely nothing.
You deserve a medal for each time shit like this doesn't get into upstream